### ISSUE TITLE
Feature Request: Asset Folders & Project Folders on the Website

### ISSUE THREAD

**Issue by Erelynn** [photopea/photopea#4189]
Photopea is amazing, but there is one thing I really wish was possible. I would like to create folders in Photopea with files that I can access online just by logging in. These folders could contain assets, such as graphics. Often, I need to reuse the same graphics over and over and I wish these graphics would be available for me to access even when I'm not using my own computer. The folders may also contain finished (or in progress) projects that I may want to keep there until I decide to save them on my personal computer. When I'm not at home, I might not be using my own computer, so I don't have access to my files or I just don't want to download my files on a computer that isn't my own. I would be okay working with my files though if I could maintain an online workflow. One way that I think this could be made possible is if Photopea gives users the option to save some of their graphics on the website, and organize them into folders.

I would like to quickly get my files into a folder by dragging & dropping them there all at once, instead of one by one. This way I can quickly upload a whole folder from my computer by dragging all of the files it contains into a folder on Photopea, which I would be able to access online. Canva has this feature and I think it’s very useful to be able to quickly upload my graphics there, and then organize them into folders.

I like that I can use Photopea without needing to make an account, but I wish there was an option to sign in simply for storing files. The login should be totally optional for anyone who wants this feature, and users should still be able to do their work without being required to sign in to an account.

I understand that storage is not free, and if Photopea does provide some storage to users on the website it will likely come at a price. I’m not sure how purchasing storage would work, but I would buy it if it was a one-time purchase. Again I don’t know if a one-time purchase option exists for online storage, but I would be interested if that option exists, and I would even pay a higher price for it. I would just prefer some private payment options, such as PayPal and gift cards, since it’s good to have these options for people who don’t like to put their credit card info online.

Also, I would like to suggest an option to donate to Photopea to support its development. I can see myself donating to support a program that I like whenever I have the ability to do so, as long as I’m not obligated to sign up for any subscriptions. I really like Photopea and it is a program that I’d want to support because I want it to become as great as possible.

Thank you for reading my feature suggestion! :)

**Comment by klade-png** [comment]
I like but I feel like it should a premium feature since it will further help development.

**Comment by Erelynn** [comment]
I wish there was some kind of option to maintain an online workflow when handling files.

In the past, I have suggested a feature that would allow users to import files into Photopea directly from Gmail or Google Drive, and then save the finished projects directly into their Email or Google Drive without having to save them to their computer first. I thought this was good because it wouldn't require any storage from Photopea. Yet maybe the downside of it could be some privacy concerns that people had.

I thought a workaround to this would be if Photopea itself offered a little bit of storage to users, as I've suggested in this post. This method might provide a little more privacy for users who are concerned about it.

I just wish there are some kind of option for maintaining an online workflow, regardless of the way it's implemented.

As far as it being a premium feature, I think it's a good thing to support the developers. I can't see myself paying for any subscriptions though. I would rather just pay for the amount of storage that I need and support development by donating whenever I can.

These are just my ideas on features that might be useful for an online workflow, but I'd love to hear of any other ideas that could work better.
